metrics-api/src/main/java/software/amazon/swage/metrics/MetricContext.java

```diff
@@ -73,6 +72,25 @@ default void count(Metric label) {
         count(label, 1L);
     }
 
+    /**
+     * Create a child context with the supplied attributes.
+     * The child context will inherited the attributes from this instance and
+     * use the supplied attributes as overrides.
```

Comment:
This would probably be better phrased as something like "The child context will contain the provided attributes as well as inherit the attributes from this instance.  Where the provided attributes conflict, they will override the values from this parent.

Also:  Can you be explicit about the relationship between parent/child here?  Is the child still related to the parent in any way, or are they completely separate things once created?  This needs to be defined at the interface level, or else different implementations may make different, incompatible choices.